MM2RandoLib/Randomizers/Enemies/REnemies.cs

```diff
@@ -87,14 +87,18 @@ private void Execute(Patch Patch, Random r)
             {
                 // Skip processing the room if every sprite bank row is taken
                 if (sbrg.IsSpriteRestricted && sbrg.SpriteBankRowsRestriction.Count >= 6)
+                {
```

Comment:
I agree, and normally I would put those into separate PRs.  I had set my compiler settings to make some of these errors, so it would fail to compile.  I promise to put future non-functional changes into separate PRs.